src/main/java/io/codelex/enums/practice/ScissorPaperStone.java

```diff
@@ -0,0 +1,66 @@
+package io.codelex.enums.practice;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class ScissorPaperStone {
+    public static void main(String[] args) {
+
+        Scanner input = new Scanner(System.in);
+        int wins = 0;
+        int losses = 0;
+        int trials = 0;
+        System.out.println("Let us begin...");
+
+        while (true) {
+            System.out.println();
+            System.out.println("Scissor-Paper-Stone");
+            System.out.print("Your turn (Enter s for scissor, p for paper, t for stone, q to quit): ");
+            String playerMove = input.nextLine();
+
+            if (playerMove.equals("q")) {
+                System.out.println("Number of trials: " + trials);
+                System.out.println("I won: " + losses);
+                System.out.println("You won: " + wins);
+                System.out.println("Bye!");
+                break;
+            }
+
+            if (playerMove.equals(Move.STONE.getValue()) || playerMove.equals(Move.PAPER.getValue()) || playerMove.equals(Move.SCISSOR.getValue())) {
+                trials++;
+                String computerMove = getComputerMove();
+
+                if (playerMove.equals(computerMove)) {
+                    System.out.println("My turn: " + computerMove);
+                    System.out.println("Tie!");
+                } else if (isWinnerPlayer(playerMove, computerMove)) {
+                    System.out.println("My turn: " + computerMove);
+
+                    System.out.println("You won!");
+                    wins++;
+                } else {
+                    System.out.println("My turn: " + computerMove);
+
+                    System.out.println("You lost!");
+                    losses++;
+                }
+
+            } else {
+                System.out.println("Invalid input, try again...");
+            }
+        }
+
+    }
+
+    private static String getComputerMove() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(3);
+        return Move.values()[randomNumber].getValue();
```

Comment:
šo metodes daļu arī varētu pašā enum klasē ielikt `getRandomValue()` for example

Reply:
šo īsti nesapratu kā tas domāts..

Reply:
Tas droši vien bija domāts, ka Move enumeratorā iebūvēt funkcionalitāti, kura atgriež random vērtību. Bet var palikt kā ir!

Reply:
`public enum Move {
    SCISSOR("s"),
    PAPER("p"),
    STONE("t");

    private static final Move[] moves = values();
    private static final int size = moves.length;
    private static final Random random = new Random();
    private final String value;

    Move(String value) {
        this.value = value;
    }

    public static String getRandomValue() {
        return moves[random.nextInt(size)].getValue();
    }

    public String getValue() {
        return value;
    }

}`

Kaut kā šādi tas getRandomValue() bija domāts?